### PR TITLE
Met 29 starting documentation for service

### PR DESCRIPTION
**Pull request by johnnolan**
# Purpose

This PR aims to put in place the beginnings of the documentation for this service.

Fixes Ticket: MET-29

## Approach

Followed the new OPG README guidelines as a reference to best add enough documentation for people to contribute and use the service.

## Checklist

* [ ] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by johnnolan** [comment]
Note for me, squash and merge this PR

File: ABOUT_THIS_SERVICE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# About this service
+
+## Mission Statement
+
+OPG has many 'things' to measure to aid in decision making, but these are not visible to all or accessible in one place, making it difficult to see patterns and connections, or high level problems across products.
+
+## Problems this service aims to solve
+
+* A common language that can be used across teams and OPG.
+* Encourage and Gamify the culture of celebrating maintenance and managing legacy
```
Comment (mattmachell) [inline ABOUT_THIS_SERVICE.md:10]: Not sure Gamify is an aim.

File: ABOUT_THIS_SERVICE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+* Understanding our users over time
+* Hold ourselves accountable throughout the entire service life-cycle
+* Think about performance and monitoring at the start and what done/good looks like
+* Reduce barriers to entry for getting data
+
+## How should you use this service
+
+This service is not intended to be a place where you debug issues or store logs that you can diagnose problems. It is to enable you to pass key bits of information across multiple services so you can overlay data points, find patterns and maintain visibility of your service.
```
Comment (mattmachell) [inline ABOUT_THIS_SERVICE.md:18]: Feel like it needs something about how it should be comprehensible to non-technical staff to make decisions or kickstart investigations..

File: LICENCE [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Ministry of Justice
```
Comment (mattmachell) [inline LICENCE:3]: 2021

**Review by mattmachell — CHANGES_REQUESTED** [review]
See inline comments for a few suggestions.

File: ABOUT_THIS_SERVICE.md [hunk 4]
```diff
@@ -17,4 +17,4 @@ OPG has many 'things' to measure to aid in decision making, but these are not vi
 
 This service is not intended to be a place where you debug issues or store logs that you can diagnose problems. It is to enable you to pass key bits of information across multiple services so you can overlay data points, find patterns and maintain visibility of your service.
 
-If you do find something happening you should use other tools that are built for this in mind to dig deeper into the issue.
+The end result should be a visualisation that will enable all professions to be able to identify changes in the service and act together.
```
Comment (mattmachell) [inline ABOUT_THIS_SERVICE.md:20]: Top phrasing! 👍

**Review by mattmachell — APPROVED** [review]

**Review by mattmachell — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,29 +1,90 @@
...
+
+Building a new metrics service for OPG.
+
+To find out more about this service, see our [about this service](ABOUT_THIS_SERVICE.md) documentation.
+
+## Prerequisites
+
+aws-vault for managing AWS access.
+
+`brew install --cask aws-vault`
```
Comment (charlesmmarshall) [inline README.md:17]: Is it worth linking out to https://docs.opg.service.justice.gov.uk/documentation/get_started.html#get-started

**Review by charlesmmarshall — APPROVED** [review]

**Review by charlesmmarshall — APPROVED** [review]

**Pull request merged**